test/OllamaApiClientTests.cs

```diff
@@ -17,22 +17,53 @@ public class OllamaApiClientTests
 {
 	private OllamaApiClient _client;
 	private HttpResponseMessage? _response;
+	private Dictionary<string, string>? _expectedRequestHeaders;
 
 	[OneTimeSetUp]
-	public void Setup()
+	public void OneTimeSetUp()
 	{
 		var mockHandler = new Mock<HttpMessageHandler>(MockBehavior.Strict);
 
 		mockHandler
 			.Protected()
 			.Setup<Task<HttpResponseMessage?>>(
 				"SendAsync",
-				ItExpr.Is<HttpRequestMessage>(_ => true),
+				ItExpr.Is<HttpRequestMessage>(r => ValidateExpectedRequestHeaders(r)),
 				ItExpr.IsAny<CancellationToken>())
 			.ReturnsAsync(() => _response);
 
 		var httpClient = new HttpClient(mockHandler.Object) { BaseAddress = new Uri("http://empty") };
 		_client = new OllamaApiClient(httpClient);
+
+		_client.DefaultRequestHeaders["required_default_header"] = "ok";
```

Comment:
This is how default request headers can be set which are always sent with requests to the Ollama API